code/modules/world_api/commands/server_query.dm

```diff
@@ -110,7 +110,7 @@
 /datum/topic_command/get_count_mod/run_command(queryparams)
 	var/n = 0
 	for (var/client/client in clients)
-		if (client.holder && (client.holder.rights & R_MOD) && !(client.holder.rights & R_ADMIN))
+		if (client.holder && client.holder.rights == R_MOD)
```

Comment:
Mods have ban aswell. So that counts the trials

Reply:
Addressed.